extensions/coap/nanofi/coap_functions.c

```diff
@@ -115,6 +121,7 @@ int create_endpoint_context(coap_context_t **ctx, const char *node, const char *
 }
 
 struct coap_pdu_t *create_request(struct coap_context_t *ctx, struct coap_session_t *session, coap_optlist_t **optlist, unsigned char code, coap_str_const_t *ptr) {
+  UNUSED(ctx);
```

Comment:
If you don't use it, I think its better not to name it instead.

Reply:
Unfortunately this part is C code, and C99 standard does not allow parameters without names, otherwise you get a `parameter name omitted` error:

[6.9.1.5] If the declarator includes a parameter type list, the declaration of each parameter shall include an identifier, except for the special case of a parameter list consisting of a single parameter of type void, in which case there shall not be an identifier. No declaration list shall follow.